py-bns.py

```diff
@@ -58,6 +71,13 @@ def connect(self):
         # save access token for later use
         self.access_token = str(response.json()['access_token'])
 
+    def get_story(self):
+        # story id
+        story_id = 'P6ZNX36JIJUP'
+        # request the json from this url with the story id at the end
+        r = self.get('syndication/stage/portal/adapter/v1/api/v1/articles/{0}'.format(story_id))
+        print(r.json())
```

Comment:
let's make this a simple return statement. It's up to the user's application what gets done to the story data